### PR TITLE
Document the backup process

### PR DESCRIPTION
**Pull request by autra**
This commit adds a short documentation detailing how to backup one's instance.

It looks obvious, but I'm still wondering whether I forgot something or not. This PR is a mean to answer this question for me and others in the future.

Thanks!

### CODE REVIEW

**Comment by autra** [comment]
Aaaannnd I guess I forgot about the configuration file

**Comment by ccoenen** [comment]
Thanks for the writeup :-) that is certainly something that has come up every now and then and which we _should_ document.

I would like to suggest giving it its own file in `docs/guides/backup.md`.

File: README.md [hunk 1]
```diff
@@ -66,6 +66,16 @@ To use CodiMD, your browser should match or exceed these versions:
...
 
+## Backup/restore your instance
+
+To backup codimd, you should:
+
+- backup your database
+- backup your custom config file if you have one
+- backup the upload folder (see the [uploadsPath](./docs/configuration.md#codimd-paths-stuff) config directive) 
+
+Restoring an existing instance of codimd is then just a matter of restoring these two things.
```
Comment (InnayTool) [inline README.md:77]: I think it has to be "three" things. ;)

**Review by InnayTool — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (autra) [reply README.md]: You're right! I removed the quantifier, it will scale better ;-)

I'm still not entirely happy with this sentence though...

Comment (InnayTool) [reply README.md]: I think it's okay like that.

**Review by InnayTool — APPROVED** [review]

**Comment by autra** [comment]
> I would like to suggest giving it its own file in `docs/guides/backup.md`.

I don't know... For me it's about the same importance as the configuration section for instance, and I felt it was overkill to create a new page and link it here for such a small section. What do you think ?

**Comment by ErikMichelson** [comment]
Although I like the idea of having a separate page, I'm also fine with the integration into the readme page as the section really isn't too long.

**Comment by ccoenen** [comment]
Ok, fine as well :-)

**Comment by autra** [comment]
Thanks!

**Pull request merged**